packages/cli/src/lib/configureStorybook.js

```diff
@@ -0,0 +1,68 @@
+import fs from 'fs-extra'
+
+import { getPaths } from '.'
+
+/**
+ * Configure Storybook for the given template by creating a custom preview config
+ */
+export default function configureStorybook({ force }, newStorybookPreview) {
+  const { storybookPreviewConfig } = getPaths().web
+
+  let finalNewStorybookPreview
```

Comment:
```suggestion
  let storybookPreviewConfig
```

I come from a Java background (a long time ago...) where `final` means `const`. So having `final` at the start of the variable name made it look a little bit like a variable that shouldn't be changed. 

Plus, now (if you accept my other suggestions), `storybookPreviewConfig` is free to use for the actual config file content, not the path.

Reply:
That make sense